script.myepisodes/default.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import threading
+import xbmc
+import xbmcaddon
+
+__addon__         = xbmcaddon.Addon()
```

Comment:
in the future please do not use \__varname\__ type labels. These are reserved for python internals.